mods/ra/rules/vehicles.yaml

```diff
@@ -563,7 +563,7 @@ TTNK:
 	Armament:
 		Weapon: TTankZap
 		LocalOffset: 0,0,213
-	AttackFrontal:
+	AttackOmni:
```

Comment:
Yes, that works. Will make the ttnks be able to also shoot on the move, though.

Just add this, ROT is 255 already by default:
```
Turreted:
AttackTurreted:
```